src/basic/points.js

```diff
@@ -114,10 +118,10 @@ Point.getDraw = function(c3ss, id) {
 
 	if (checkMarkerSym(c3ss)) {
 
-		Object.assign(
-				point,
-				getColors(c3ss),
-				getWidths(c3ss)
+		point = mergeWith(
```

Comment:
Just asking: What are the benefits of using `mergeWith/merge` instead of de-facto function `Object.assign`?